server/routes/index.js

```diff
@@ -1,13 +1,20 @@
 const express = require('express');
+const updateVoterStatus = require('../database/queries/updateVoterStatus');
 const path = require('path');
-
 const router = express.Router();
 const voters = require('../model/voters');
 /* GET home page. */
 
 router.get('/voters', (req, res) => {
   res.json(voters);
 });
+router.get('/idVoter/:id/status/:status', (req, res) => {
+  // UpdateStatus
+  updateVoterStatus(req.params.id, req.params.status, (err) => {
+    // cb to handle errors
+    err ? res.status(500).end() : res.status(200).end();
```

Comment:
Please fix your linting errors in this file (line 3 and 15). In general, eslint doesn't like ternary operators so you might have to check the eslint documentation to fix this one and maybe disable this rule in your configuration.

Reply:
fixed :+1:  
[link](https://github.com/eslint/eslint/issues/7469) that helped me

Reply:
https://github.com/eslint/eslint/issues/7469#issuecomment-257009766